tests/python/relay/aot/corstone300.mk

```
@@ -24,8 +24,16 @@ CRT_ROOT ?= ${TVM_ROOT}/build/standalone_crt
 ifeq ($(shell ls -lhd $(CRT_ROOT)),)
 $(error "CRT not found. Ensure you have built the standalone_crt target and try again")
 endif
+
 FVP_DIR ?= /opt/arm/FVP_Corstone_SSE-300_Ethos-U55/models/Linux64_GCC-6.4/
 
+NPU_MACS ?= 256
+
+MODEL = FVP_Corstone_SSE-300_Ethos-U55
+ifeq (${NPU_VARIANT}, U65)
+MODEL = FVP_Corstone_SSE-300_Ethos-U65
+endif
```

Could this just append the variant?

```suggestion
NPU_MACS ?= 256
NPU_VARIANT ?= U55

MODEL = FVP_Corstone_SSE-300_Ethos-$(NPU_VARIANT)
```

---

Thanks for the suggestion @Mousius, that makes much more sense indeed!